wgpu-core/src/command/compute.rs

```diff
@@ -919,9 +919,17 @@ pub mod compute_ffi {
             return;
         }
 
+        // Clamp the number of dynamic offsets to the maximum value that the
```

Comment:
For validation we should include the original length, so that in the body of the pass we can check to make sure the original_length == clamped length, and error if they don't match.